crates/cli/Cargo.toml

```diff
@@ -9,9 +10,9 @@ description = "Trdelník is Rust based testing framework providing several conve
 
 [dependencies]
 clap = { version = "3.1.6", features = ["derive"] }
-tokio = { version = "1.17.0", features = ["rt-multi-thread", "macros", "fs", "signal", "sync", "time", "io-util", "process"], default-features = false }
+tokio = { version = "~1.14.1", features = ["rt-multi-thread", "macros", "fs", "signal", "sync", "time", "io-util", "process"], default-features = false }
```

Comment:
What updates block us from upgrading beyond `1.15`?

Reply:
Solana SDK, and we can't bump Solana SDK because of Anchor.